### DISCUSSION TITLE
Running audiamus via Parallels Desktop on M1 Mac

### DISCUSSION THREAD

**Discussion by delta0815** [audiamus/AaxAudioConverter#182] [Q&A]
Dear all,

I'm a huge fan of Audiamus and I've been using it for quite some time now. However I recently moved to the Apple world. There are just a few applications which are only available on Windows and audiamus is one of them.

I was able to run every native Window app via Parallels Desktop as it seems to emulate X86 code in a Window ARM environment. The audiamus installer however refuses to install the app.

I'm a noob in that space but from what I've read a native X86 app should still work on an ARM version of Windows, so could it be possible that if the setup.exe would let me install audiamus it would work? And if so would it be possible to disable the X86 check in the installer.

Thanks for your thoughts.

Cheers,

Oliver

**Comment by audiamus** [comment]
The current version of AAX Audio Converter is built on .Net Framework 4.8. To my knowledge this supports Windows on x86/x64 only. The plan is to migrate to .Net 6 (Core) later this year. It's more than just a simple upgrade and I also want to include the still separate app Book Lib Connect. With .Net 6, some support for Windows on ARM appears to be available, emulation or native, but I don't know whether this extends to Windows Form, the GUI framework in use and to remain in use. And AAX Audio Converter will continue to use some other Windows specific APIs. 

If the installer does not work on your experimental platform, you can install to a proper Windows machine, copy-over all the installed files and try with those. However, I am not able to provide further assistance with such attempts.
